components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/idp/sso/configurator/IS500SAMLSSOConfigurator.java

```diff
@@ -188,13 +197,22 @@ public boolean updateProvider(SSOProvider provider) {
 
     @Override
     public boolean updateProvider(WebApp application) {
+        String acsUrl = application.getAcsURL().trim();
         SSOProvider ssoProvider = application.getSsoProviderDetails();
         if (ssoProvider == null) {
             log.warn("No SSO Configurator details given. Manual setup of SSO Provider required.");
         }
 
         //ssoProvider.setIssuerName(app.getId().getApiName());
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/f3131fd198c08449ff724321a95ba67682999cc3